### ISSUE TITLE
uninitialised constant (Stellar::Currency)

### ISSUE THREAD

**Issue by brahman81** [astroband/ruby-stellar-sdk#3]
When creating a Stellar::Client object, we receive an uninitialised constant error, I have been unable to find any definition of the Stellar::Currency class in either the stellar-base or stellar-sdk.

In this specific case we are expecting a Stellar::Currency.native() method.

```ruby
require 'stellar-sdk'
client = Stellar::Client.localhost()
```

```
/var/lib/gems/2.3.0/gems/stellar-sdk-0.1.1/lib/stellar/amount.rb:8:in `<class:Amount>': uninitialized constant Stellar::Amount::Currency (NameError)
        from /var/lib/gems/2.3.0/gems/stellar-sdk-0.1.1/lib/stellar/amount.rb:2:in `<module:Stellar>'
        from /var/lib/gems/2.3.0/gems/stellar-sdk-0.1.1/lib/stellar/amount.rb:1:in `<top (required)>'
        from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
        from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
        from /var/lib/gems/2.3.0/gems/backports-3.6.8/lib/backports/std_lib.rb:9:in `require_with_backports'
        from /var/lib/gems/2.3.0/gems/stellar-sdk-0.1.1/lib/stellar/client.rb:59:in `<class:Client>'
        from /var/lib/gems/2.3.0/gems/stellar-sdk-0.1.1/lib/stellar/client.rb:4:in `<module:Stellar>'
        from /var/lib/gems/2.3.0/gems/stellar-sdk-0.1.1/lib/stellar/client.rb:3:in `<top (required)>'
        from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
        from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
        from /var/lib/gems/2.3.0/gems/backports-3.6.8/lib/backports/std_lib.rb:9:in `require_with_backports'
        from create_account.rb:24:in `<main>'
```

Is this genuine or am I missing something obvious?

**Comment by nullstyle** [comment]
This looks genuine...  We changes the name of `Currency` to `Asset` quite a while back, and I'm sure I didn't update this library.  I'll look into fixing it now.

**Comment by nullstyle** [comment]
Fixed in a622c920ed61fb9ad8274ecdafbb75d8e02501e9
